### PR TITLE
Minor update about REST and AutoDV2 support to Exchange-Teams-interact.md

### PR DESCRIPTION
**Pull request by dariomws**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/Exchange-Teams-interact.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The following table provides a helpful quick reference to feature availability b
 
 <sup>7</sup> Teams honors the [Outlook on the web mailbox policy](https://docs.microsoft.com/powershell/module/exchange/client-access/set-owamailboxpolicy) setting that's configured by tenant admins to control whether users can change their profile picture. If the **-SetPhotoEnabled** setting is turned off in the policy, users cannot add, change, or remove their profile picture. For example, if a user uploads a profile picture that's approved by your organization's IT or HR department, no action is needed. However, if a user uploads an inappropriate picture, change it according to your organization's internal policies.
 
-<sup>8</sup> You need to meet the requirements listed on [Requirements for mailboxes hosted on-premises](#requirements-for-mailboxes-hosted-on-premises) section.
+<sup>8</sup> You need to meet the requirements listed on [Requirements to create and view meetings for mailboxes hosted on-premises](#Requirements-to-create-and-view-meetings-for-mailboxes-hosted-on-premises) section.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:64]:
```suggestion
<sup>8</sup> You need to meet the requirements listed in the [Requirements to create and view meetings for mailboxes hosted on-premises](#Requirements-to-create-and-view-meetings-for-mailboxes-hosted-on-premises) section.
```

File: Teams/Exchange-Teams-interact.md [hunk 2]
```diff
@@ -78,20 +78,20 @@ Microsoft Teams works with several Microsoft 365 and Office 365 services to prov
 > [!IMPORTANT]
 > If you uninstall the Skype for Business client after you move a user to **Teams Only** mode, presence may stop working in Outlook and other Office apps. Presence works fine in Teams. To resolve this issue, select your profile picture in the top right-hand corner of Microsoft Teams and then select **Settings**. On the **General** tab under **Application**, select **Register Teams as the chat app for Office (requires restarting Office applications)**. After you select this option, close and re-open all Office apps, including Outlook. After you open Outlook, presence information will be available.
 
-## Requirements for mailboxes hosted on-premises
+## Requirements to create and view meetings for mailboxes hosted on-premises
 
-If users want the capability to schedule a Teams meeting using Exchange Server on-premises, the following requirements must be meet:
+If mailboxes are hosted on-premises, to create and view meetings, the following requirements must be meet:
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:83]:
```suggestion
If mailboxes are hosted on-premises, to create and view meetings, the following requirements must be met:
```

File: Teams/Exchange-Teams-interact.md [hunk 3]
```diff
@@ -78,20 +78,20 @@ Microsoft Teams works with several Microsoft 365 and Office 365 services to prov
...
 
 - Users must be synchronized to Azure Active Directory. For information about how to use Azure AD Connect to synchronize with Azure Active Directory, see [Hybrid identity documentation](https://docs.microsoft.com/azure/active-directory/hybrid/).
 
 - Mailboxes are hosted in Exchange Server 2016 Cumulative Update 3 or later.
 
-- Autodiscover and Exchange Web Services is published externally.
- 
 > [!NOTE]
-> Autodiscover (AutoD) V2 is required to allow the Teams service to perform an unauthenticated discovery of the user's mailbox. AutoD V2 is supported in Exchange 2016 CU3 and later.
+> Exchange 2016 CU3 adds support for REST API integration with Microsoft 365 and Autodiscover (AutoD) V2. REST is required to support Calendar application in Teams, for more information, see [Use REST APIs to access mailboxes in Exchange hybrid deployments](https://docs.microsoft.com/graph/hybrid-rest-support). Autodiscover (AutoD) V2 is required to allow the Teams service to perform an unauthenticated discovery of the user's mailbox, and also to support scheduling in Outlook for Teams.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:92]:
```suggestion
> Exchange 2016 CU3 adds support for REST API integration with Microsoft 365 and Autodiscover (AutoD) V2. REST is required to support the Calendar application in Teams. For more information, see [Use REST APIs to access mailboxes in Exchange hybrid deployments](https://docs.microsoft.com/graph/hybrid-rest-support). Autodiscover (AutoD) V2 is required to allow the Teams service to perform an unauthenticated discovery of the user's mailbox, and also to support scheduling in Outlook for Teams. 
```

File: Teams/Exchange-Teams-interact.md [hunk 4]
```diff
@@ -78,20 +78,20 @@ Microsoft Teams works with several Microsoft 365 and Office 365 services to prov
...
 
 - Mailboxes are hosted in Exchange Server 2016 Cumulative Update 3 or later.
 
-- Autodiscover and Exchange Web Services is published externally.
- 
 > [!NOTE]
-> Autodiscover (AutoD) V2 is required to allow the Teams service to perform an unauthenticated discovery of the user's mailbox. AutoD V2 is supported in Exchange 2016 CU3 and later.
+> Exchange 2016 CU3 adds support for REST API integration with Microsoft 365 and Autodiscover (AutoD) V2. REST is required to support Calendar application in Teams, for more information, see [Use REST APIs to access mailboxes in Exchange hybrid deployments](https://docs.microsoft.com/graph/hybrid-rest-support). Autodiscover (AutoD) V2 is required to allow the Teams service to perform an unauthenticated discovery of the user's mailbox, and also to support scheduling in Outlook for Teams. 
+
+- Autodiscover and Exchange Web Services is published externally.
```
Comment (JohanFreelancer9) [inline Teams/Exchange-Teams-interact.md:94]:
```suggestion
- Autodiscover and Exchange Web Services are published externally.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by dariomws** [comment]
Replaced by https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/pull/5680.

**Pull request closed without merging**